docs/en/query_language/functions/bit_functions.md

```diff
@@ -198,5 +198,36 @@ Result:
 │                    0 │
 └──────────────────────┘
 ```
+## bitCount {#bitcount}
 
+Calculates the number of bits set in the [integer](../../data_types/int_uint.md) number.
```

Comment:
```suggestion
Calculates the number of bits set to one in the [integer](../../data_types/int_uint.md) number.
```

Reply:
Also, this function should support floating point types.

Reply:
You're right, I added it in the new version.